src/lib/Scenes/Home/Components/Auctions/index.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from "react"
+import { FlatList, SectionList, TouchableWithoutFeedback, View } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components/native"
+
+import fonts from "../../../../../data/fonts"
+import { AuctionItem } from "./Components/AuctionItem"
+
+const Container = styled.View`
+  flex: 1;
+  padding: 10px 15px;
+`
+
+const SectionHeader = styled.View`
+  padding-top: 15px;
+  padding-bottom: 10px;
+  background-color: white;
+`
+
+const SectionTitle = styled.Text`
+  font-family: ${fonts["garamond-regular"]};
+  font-size: 25px;
+  text-align: left;
+  margin-left: 2px;
+`
+
+interface Props {
+  auctions: Array<{
+    live_start_at: string | null
+  }>
+}
+
+class Auctions extends React.Component<Props, null> {
+  handleTap(item) {
+    console.log(item)
+  }
+
+  renderList(itemData) {
+    return (
+      <FlatList
+        contentContainerStyle={{ justifyContent: "space-between", padding: 5, display: "flex" }}
+        data={itemData.data}
+        numColumns={2}
+        keyExtractor={(item, index) => item.id}
+        renderItem={d => {
+          return (
+            <TouchableWithoutFeedback onPress={this.handleTap.bind(this, d)}>
+              <View style={{ marginRight: 10, marginBottom: 10 }}>
+                <AuctionItem key={d.index} auction={d.item} />
+              </View>
+            </TouchableWithoutFeedback>
+          )
+        }}
+      />
+    )
+  }
+
+  render() {
+    const auctions = this.props.auctions
+    const liveAuctions = auctions.filter(a => !!a.live_start_at)
+    const timedAuctions = auctions.filter(a => !a.live_start_at)
+    const sections = [
+      {
+        data: [
+          {
+            data: liveAuctions,
+          },
+        ],
+        title: "Current Live Auctions",
+      },
+      {
+        data: [
+          {
+            data: timedAuctions,
+          },
+        ],
+        title: "Current Timed Auctions",
+      },
+    ]
+
+    return (
+      <SectionList
+        contentContainerStyle={{ justifyContent: "space-between", padding: 15, display: "flex" }}
+        stickySectionHeadersEnabled={false}
+        sections={sections}
+        keyExtractor={(item, index) => item.id}
+        renderItem={itemData => {
+          return this.renderList(itemData.item)
+        }}
+        renderSectionHeader={({ section }) =>
+          <SectionHeader>
+            <SectionTitle>
+              {section.title}
+            </SectionTitle>
+          </SectionHeader>}
+      />
+    )
+  }
+}
+
+export default createFragmentContainer(Auctions, {
+  auctions: graphql`
+    fragment Auctions_auctions on Sale @relay(plural: true) {
+      ...AuctionItem_auction @relay(mask: false)
```

Comment:
this is an interesting approach 👍

Reply:
Yeah what made you go for that? :) I read the docs and saw the example they have. Does this mean every time an auction for an auction item is fetched, we get all the fields? or always a set of specific fields? (although those would then also be defined here right?)

Reply:
Relay traditionally hides props you don't ask for, this says "give this object the whole thing" - now that I think about this a second time it could probably be:

```js
fragment Auctions_auctions on Sale @relay(plural: true) {
      ...AuctionItem_auction @relay()
     live_start_at
}
```

and you'd get the same behavior - _I think_ have not read the docs nor used Really Modern though 👯‍♂️

Reply:
@mennenia As @orta mentioned, Relay usually hides props you don't specifically ask for.
From the docs
> While typically Relay only provides the data for fields explicitly requested by a component's fragment, `@relay(mask: false)` can be added to a fragment spread to not mask that data, recursively including the data from the fields of the referenced fragment.

When `@relay(mask: false)` is used, you only get the fields specified in the child fragment but they're unmasked so you have access to the properties in the parent component. That also means you can use a regular React component instead of relay fragments for the item component.

However, the following option is also valid.
```
fragment Auctions_auctions on Sale @relay(plural: true) {
      ...AuctionItem_auction
     live_start_at
}
```
Since we only need `live_start_at` in the filter, I'll use that instead to avoid confusion as it's the pattern we've been using up until now.